### PR TITLE
Add unit test for owner_info (RIPD-1416)

### PR DESCRIPTION
**Pull request by mellery451**
This RPC method is deprecated, but i've added a basic test to cover it
until we actually remove it.

### CODE REVIEW

**Comment by mellery451** [comment]
sure - I'll look into adding an offer or two

File: src/test/rpc/OwnerInfo_test.cpp [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+        env(pay(alice, gw, CNY(50)));
+        env(offer(alice, CNY(2), XRP(1000)));
+
+        Json::Value params;
+        params[jss::account] = alice.human();
+        auto const result = env.rpc ("json", "owner_info",
+            boost::lexical_cast<std::string>(params)) [jss::result];
+        if (! BEAST_EXPECT (
+            result.isMember(jss::accepted) &&
+            result.isMember(jss::current)))
```
Comment (wilsonianb) [inline src/test/rpc/OwnerInfo_test.cpp]: This check makes lines 115 and 160 redundant

Comment (mellery451) [inline src/test/rpc/OwnerInfo_test.cpp]: will fix

File: src/test/rpc/OwnerInfo_test.cpp [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+            BEAST_EXPECT (result[jss::error] == "invalidParams");
+            BEAST_EXPECT (result[jss::error_message] ==
+                "Missing field 'account'.");
+        }
+
+        { // ask for empty account
+            Json::Value params;
+            params[jss::account] = "";
+            auto const result = env.rpc ("json", "owner_info",
+                boost::lexical_cast<std::string>(params)) [jss::result];
```
Comment (ximinez) [inline src/test/rpc/OwnerInfo_test.cpp]: So much simpler to just use `to_string(params)` here and the other places you use `lexical_cast`.

**Comment by seelabs** [comment]
Merged into 0.80.0-b1

**Pull request closed without merging**